_blocks/develop.md

```diff
@@ -2,7 +2,7 @@
 lang: nl
 index: 2
 title: Ontwikkel mee
-subtitle: De app is open source en je kunt de ontwikkeling volgen op Github.
+subtitle: De broncode van de app is publiekelijk beschikbaar en je kunt de ontwikkeling volgen op Github.
```

Comment:
This is not a minor change :wink:, this would deviate from the `open source` mentioned in [the timeline (Dutch)](https://www.rijksoverheid.nl/onderwerpen/coronavirus-app/tijdpad-proces-coronavirus-app).

@EdoPlantinga any changes on the open/public source definition of the app?

Reply:
Personally I prefer the original. "Publiekelijk beschikbaar" sounds a bit distant to me, I think open source is a term that is also becoming more widely understood by non-techies, plus, considering @bwbroersma 's comment it is worth keeping the term for techies.

Reply:
> this would deviate from the `open source` mentioned in [the timeline (Dutch)](https://www.rijksoverheid.nl/onderwerpen/coronavirus-app/tijdpad-proces-coronavirus-app).

The timeline mentions “open source _software_”, which hints at the definition of OSS, as per OSI. The way it was used here–i.e. only mentioning the concept of source being available to review–and how the rest of the copy is phrased, it seemed to me like the target audience wasn’t necessarily tech-savvy people.

Seeing as this is the case, how about changing it to “open-source software”?

Reply:
Agree with "open source software". Please note @EdoPlantinga has the official <span lang=nl>"Open source community management"</span> behind his name in the same document, that hints more to open source development. Although I agree that it's not open source to the fullest, it's nice to at least keep it in to keep the government on the right track with the name :wink:.

Reply:
Done